src/libDirectoryService/FinalBlockPreProcessing.cpp

```diff
@@ -1154,6 +1157,45 @@ void DirectoryService::RunConsensusOnFinalBlock() {
     LOG_GENERAL(INFO, "RunConsensusOnFinalBlock ");
     PrepareRunConsensusOnFinalBlockNormal();
 
+    m_microblock_gas_limit = MICROBLOCK_GAS_LIMIT;
```

Comment:
Add a comment above to talk about the logic of exponential backoff

Reply:
Added